src/jsxstyle-components.js

```diff
@@ -0,0 +1,9 @@
+// Mostly copied from:
+// https://github.com/threepointone/glamor/blob/
+// 5e7d988211330b8e2fca5bb8da78e35051444efd/src/jsxstyle.js
+/*
+ * This is used to generate the helper functions
+ * like: glamorous.block and the <glamorous.Block /> elements
+ */
+
+export default ['view', 'block', 'inlineBlock', 'flex', 'row', 'column']
```

Comment:
> I think the assumption is that DOM elements are accessible directly on `glamorous`. If we start shoving other things on `glamorous` would that get confusing? Maybe not I guess.

It'd get a little cluttered. I like your idea of `.display`, I just wonder if all the other elements might be better off moved there as well.

`import { A, Flex, Row, Column } from glamorous.display`